install_config/topics/configuring_gce_objects-proc.adoc

```diff
@@ -0,0 +1,171 @@
+fine-grained roles are available using the IAM service.
++
+You can create service accounts with specific permissions, then use them to
+deploy infrastructure components instead of regular users. Also, you can create
+roles to limit access to different users or service accounts.
++
+GCE instances use service accounts to allow applications to call GCE APIs. For
+example, {product-title} node hosts can call the GCE disk API to provide a
+persistent volume to an application.
++
+Deploying {product-title} requires the proper permissions. A user must be able
+to create service accounts, cloud storage, instances, images, templates, Cloud
+DNS entries, and deploy load balancers and health checks. Delete permissions are
+also helpful in order to be able to redeploy the environment while testing.
++
+For more information, see link:https://cloud.google.com/compute/docs/access/[the Access cloud overview section of the GCE documentation].
+
+SSH keys::
+
+GCE injects SSH public keys as authorized keys to be able to log in using SSH in
+the created instances. The list of SSH keys can be configured per instance or
+per project.
+
+Configuring GCE regions and zones::
+
+GCE has a global infrastructure that covers regions and availability
+zones.
+
+[NOTE]
+====
+While deploying {product-title} in GCE on different zones can help avoid
+single-point-of-failures, but there are some caveats regarding storage. GCE
+disks are created within a zone. Therefore if a {product-title} node host goes
+down in zone "A" and the pods should be moved to zone "B", the persistent
+storage cannot be attached to those pods because the disks are in a different
+zone.
+
+See link:https://cloud.google.com/compute/docs/zones[the GCE documentation on regions and zones] or link:https://kubernetes.io/docs/admin/multiple-zones/#limitations[the Kubernetes documentation on multiple zones] for more information.
+====
+
+Internal DNS::
+
+GCE `VPC` networks have an internal DNS service that automatically
+resolves internal host names.
+
+The internal fully qualified domain name (FQDN) for an instance follows the `[HOST_NAME].c.[PROJECT_ID].internal` format.
+
+For more information, see link:https://cloud.google.com/compute/docs/internal-dns[the GCE documentation on Internal DNS] 
+
+External IP address::
+
+For GCE instances to communicate with the Internet, you must attach an external
+IP address to the instance. Also, to communicate with instances deployed in GCE
+from outside the VPC Network, an external IP address is required.
+
+[WARNING]
+====
+https://cloud.google.com/vpc/docs/vpc#internet_access_reqs[Requiring an `External IP address` for internet access] is a limitation of
+the provider. This reference architecture configures firewall rules to avoid
+incoming external traffic in instances if not needed.
+====
+
+For more information, see
+link:https://cloud.google.com/compute/docs/ip-addresses/[the GCE documentation
+on external IP address].
```

Comment:
Does attaching external IPs need more than this link? It does say it's up to the provider, but is there a specific process for GCE on OCP?